### PR TITLE
Add example repo to spring-boot-spa post

### PR DESCRIPTION
**Pull request by bdemers**
This blog post has the following features:

- [ ] A GitHub Repository with a polished README
- [ ] A GitHub Repository under the github.com/oktadev account
- [ ] A title that's approved by Dev Advocacy
- [ ] A URL that's approved by Dev Advocacy
- [ ] The content has been ran through Grammarly (https://www.grammarly.com/)
- [ ] Rendered locally and confirmed that no Markdown typos exist

### CODE REVIEW

**Comment by mraible** [comment]
I don't think a changelog is necessary for this since you're just fixing a bug rather than making a change to the article. I'd recommend adding it at the end, in the last couple of paragraphs. You could add it in the frontmatter too, but it won't show up in the GitHub icon until #908 is merged. I'm still waiting for Jimena's approval on that one.

File: _source/_posts/2021-10-04-spring-boot-spa.md
Comment (mraible) [inline _source/_posts/2021-10-04-spring-boot-spa.md]:
```suggestion
github: https://github.com/oktadev/okta-vuejs-spa-spring-boot-example
```

File: _source/_posts/2021-10-04-spring-boot-spa.md
Comment (mraible) [inline _source/_posts/2021-10-04-spring-boot-spa.md]:
```suggestion
In this tutorial, you are going to create a single-page application (SPA) that uses a Spring Boot resource server and a Vue front-end client. You'll see how to configure Spring Boot to use JSON Web Tokens (JWT) for authentication and authorization, with Okta as an OAuth 2.0 and OpenID Connect (OIDC) provider. You'll also see how to bootstrap a Vue client app with the Vue CLI and how to secure it using the Okta Sign-In Widget.
```

**Comment by mraible** [comment]
I also noticed there's no version numbers for the `npm install` command. That makes it so this post will remain more evergreen, but also more susceptible to breakage because major version upgrades might change the code that's needed.

**Comment by bdemers** [comment]
Good call, I'll pin the versions

File: _source/_posts/2021-10-04-spring-boot-spa.md
Comment (bdemers) [inline _source/_posts/2021-10-04-spring-boot-spa.md]:
```suggestion
  @okta/okta-signin-widget@5.12.0 \
  @okta/okta-vue@5.0.1 \
  @okta/okta-auth-js@5.5.0 \
  axios@0.22.0
```

**Review by mraible — APPROVED** [review]

**Pull request merged**
